Mathlib/RingTheory/Flat/Basic.lean

```diff
@@ -187,6 +187,7 @@ instance directSum (ι : Type v) (M : ι → Type w) [(i : ι) → AddCommGroup
   classical
   rw [iff_rTensor_injective]
   intro I hI
+  letI : ∀ i, AddCommGroup (↥I ⊗[R] (M i)) := fun i ↦ TensorProduct.addCommGroup
```

Comment:
This probably deserves a TODO comment, and a link to https://leanprover.zulipchat.com/#narrow/stream/144837-PR-reviews/topic/.2310828.20-.20generalizing.20CommRing.20to.20CommSemiring.20etc.2E/near/422684923

Reply:
done.